lib/components/UploadPicker.vue

```diff
@@ -140,6 +142,10 @@ export default Vue.extend({
 			type: Array as PropType<Node[]>,
 			default: () => [],
 		},
+		forbiddenCharacters: {
```

Comment:
I think we should not use a string here but an array, just like on the server backend.
Because there might be multi-byte characters in the list, which is hard to split and check otherwise.